src/components/ContactListItem/ContactListItem.js

```diff
@@ -2,10 +2,10 @@ import React from 'react'
 import styled from 'styled-components/macro'
 import { ReactComponent as UserIcon } from '../Design/img/user.svg'
 
-export default function ContactListItem({ firstName, lastName, company, id }) {
+export default function ContactListItem({ firstName, lastName, company, id, onClick }) {
   return (
     <li key={id}>
-      <ContactContainer>
+      <ContactContainer onClick={onClick}>
```

Comment:
Do you need a section inside your li or could you just move the onClick + styling to your li?